src/status_im/ui2/screens/chat/components/chat_bottom_sheet/test.cljs

```diff
@@ -0,0 +1 @@
+(ns status-im.ui2.screens.chat.components.chat-bottom-sheet.test)
```

Comment:
why empty file ?

Reply:
For component tests. Tests would be added in this file when we finish integrating RNTL into our project. Added the file now so we don't forget about it.

Reply:
I'm okay having such empty files, but I would prefer to not have them tbh. It also confuses PR reviewers.

Reply:
I have deleted the empty test files. Kept only one which you can use as an example in the conventions/project-structure documents:
```* component_name
          - view.cljs
          - style.cljs
          - test.cljs
```